content/applications/websites/ecommerce/managing_products/price_management.rst

```diff
@@ -0,0 +1,204 @@
+
+Discount and loyalty programs
+=============================
+
+It is possible to create promotion and loyalty programs, which offer different options from
+pricelists. To do so, go to :menuselection:`Website app --> eCommerce --> Discount & Loyalty`, and
+:guilabel:`Create` a program from scratch or by using the templates. You can also modify an existing
+program by clicking on it.
+
+.. image:: price_management/price-discount-loyalty.png
+   :align: center
+   :alt: Discount and loyal program templates
+
+.. _pricing_management/program-types:
+
+The different program types are:
+
+- :guilabel:`Coupons`: Generate and share single-use, coupon codes that grant access to rewards;
+- :guilabel:`Next Order Coupons`: Generate and share single-use, coupons that grant access to
+  rewards on the next order;
+- :guilabel:`Promotions`: Set conditional rules when ordering, which grant access to reward for the
+  customers;
+- :guilabel:`Discount Code`: Set codes which when entered upon checkout grants customers a discount;
+- :guilabel:`Loyalty Cards`: Customers receive points when buying, which they can exchange for
+  rewards on their current or future order;
+- :guilabel:`Buy X Get Y`: Grant 1 credit for each item bought, then reward customers with Y item in
+  exchange of X credits.
+
+Configuration
+-------------
+
+When creating or modifying a program, the options available depend on the :guilabel:`Program Type`
+selected. Therefore, some programs may have additional options related to their type.
+
+- :guilabel:`Program Name`: Enter the name of the program, this is not visible by the customer;
+- :guilabel:`Program Type`: The :ref:`program type <pricing_management/program-types>` you want to
+  use;
+- :guilabel:`Currency`: The currency you wish to use for that program;
+- :guilabel:`Points Unit`: The unit you wish to use for the :guilabel:`Loyalty Cards`. You are free
+  to enter any name you want;
+- :guilabel:`Validity`: The date until which the program is valid;
+- :guilabel:`Limit Usage`: The limited number of times the program can be used during the
+  :guilabel:`Validity`;
+- :guilabel:`Company`: If you have multiple companies, the company in which the program is
+  available;
+- :guilabel:`Available On`: Check the app boxes where the program is available on;
+- :guilabel:`Website`: The website(s) on which the program is available. Leave the field to make it
+  available on *all* websites;
+- :guilabel:`Point of Sale`: The :abbr:`PoS (Point of Sale)` in which the program is available.
+  Leave empty to make it available on *all* :abbr:`PoS (Point of Sale)`.
+
+.. image:: price_management/price-programs.png
+   :align: center
+   :alt: Program options
+
+You can see the number of promotional items available per program in the upper-right corner, under
+the corresponding tab.
+
+.. image:: price_management/price-programs-items.png
+   :align: center
+   :alt: Program items tab
+
+Conditional rules
+~~~~~~~~~~~~~~~~~
```

Comment:
```suggestion
- :guilabel:`Company`: If you have multiple companies, the company in which the program is 
available;
- :guilabel:`Available On`: Check the app boxes where the program is available on;
- :guilabel:`Website`: The website(s) on which the program is available. Leave the field to make it
  available on *all* websites;
- :guilabel:`Point of Sale`: The :abbr:`PoS (Point of Sale)` in which the program is available.
  Leave empty to make it available on *all* :abbr:`PoS (Point of Sale)`.

.. image:: price_management/price-programs.png
   :align: center
   :alt: Program options

You can see the number of promotional items available per program in the upper-right corner, under
the corresponding tab.

.. image:: price_management/price-programs-items.png
   :align: center
   :alt: Program items tab

Conditional rules
~~~~~~~~~~~~~~~~~
```

Reply:
Same here, not sure why the blank space was deleted